src/praisonai/tests/unit/llm/test_env_resolver.py

```
@@ -276,13 +276,35 @@ def test_all_providers_have_correct_base_urls(self):
             "groq/": "https://api.groq.com/openai/v1",
             "cohere/": "https://api.cohere.ai/v1",
             "openrouter/": "https://openrouter.ai/api/v1",
+            "orcarouter/": "https://api.orcarouter.ai/v1",
             "ollama/": "http://localhost:11434/v1",
         }
         
         for prefix, (key_var, base_url) in _PROVIDER_MAP.items():
```

Ruff (0.16.0)

[warning] Loop control variable `key_var` not used within loop body

Rename unused `key_var` to `_key_var`

(B007)